tekton/pipeline_run_test.go

```diff
@@ -208,7 +208,7 @@ var _ = Describe("PipelineRun", func() {
 			Expect(unmarshaledSnapshotSpec.Application).To(Equal(applicationName))
 		})
 
-		It("can add the ReleaseStrategy information to a PipelineRun object and ", func() {
+		It("can add the Strategy information to a PipelineRun object and ", func() {
```

Comment:
I liked this better as `ReleaseStrategy`. The line isn't too long and we don't have a concept of "strategy" only the release strategy

Reply:
I think that I messed it up as I refactored the status field. It changed everything else...

The only place where we are going to have `Strategy` instead of `ReleaseStrategy` is in the status itself as it will read the following way:
```
Release:
    PipelineRun: ...
    Strategy: ...
```
I'll fix those occurrences.